bin/setup

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+project=mynamespace
+
+# Check that minishift is installed
+minishift=$(which minishift)
+if [ -z "$minishift" ];
+then
+	echo "Minishift not found, see https://github.com/minishift/minishift#getting-started for installation instructions"
+	exit 1
+fi
+
+# Ensure minishift is running
+is_running=$(minishift status | grep Minishift | awk '{print $2}')
+if [ "$is_running" != "Running" ];
+then
+	echo "You must start minishift"
+	exit 1
+fi
+
+# Make sure path to the oc binary is on our PATH
+eval $(minishift oc-env)
+
+# Login as admin
+res=$(oc login -u system:admin)
```

Comment:
For some reason bash is trying to execute the stdout of every command unless I set to a variable...not sure if this is just my bash env

Reply:
That's strange ... but I also think `system:admin` was broken in a recent version of minishift. I thought we had to use the admin user addon now. Was that fixed?